media_driver/linux/common/os/mos_utilities_specific.c

```diff
@@ -1068,6 +1068,7 @@ static MOS_STATUS _UserFeature_DumpFile(const char * const szFileName, MOS_PUF_K
         }
         MOS_FreeMemory(CurKey);
     }
+    MOS_SafeFreeMemory(CurValue);
```

Comment:
is memory on CurValue not used anymore out of this function?
it seems dangerous to free, even under success return.